biot_savart_integrators/generic/line_elements.py

```diff
@@ -0,0 +1,36 @@
+"""Numerical routines for Biot-Savart integration of a generic current curve
+
+The routines expect (dask-chunked) xarray inputs in order to facilitate
+memory-efficient and parallelized computation.
+"""
+import xarray as xr
+from .integrand import biot_savart_integrand as bsintegrand
+
+def biot_savart_integral(r: xr.DataArray, r_c: xr.DataArray,
```

Comment:
Is it really necessary to create  near copy-paste of `.curve`? I thought we dicussed that you coudl simply use what is in `.curve` and just add an optional parameter `dl=None` in there so that `dl` is calculatated unless provided.

Reply:
Good point.